site/package.json

```diff
@@ -20,19 +20,22 @@
     "sync": "node ./sync.js",
     "clean": "rimraf dist"
   },
+  "engines": {
+    "node": ">=v12"
```

Comment:
We might want to specify the minor/patch version too.
This is because tools are quite ambiguous in their behavior there. `semver` interprets this as `>= 12.0.0`, but `npm` interprets this as `>= last 12.*.*`, i.e. `=== last 12.*.*`.

Reply:
Good point - fixed